packages/jest-jasmine2/src/index.js

```diff
@@ -125,12 +124,11 @@ async function jasmine2(
       environment: 'node',
       handleUncaughtExceptions: false,
       retrieveSourceMap: source => {
-        if (runtime._sourceMapRegistry[source]) {
+        const sourceMap = runtime.getSourceMapForFile(source);
```

Comment:
should probably use `runtime.getSourceMaps()` here as well and remove `getSourceMapForFile`